src/views/CoCurricularTranscript/index.tsx

```diff
@@ -54,7 +52,7 @@ const CoCurricularTranscript = () => {
                 Gordon College Experience Transcript
               </Typography>
             }
-            subheader={<SubHeader profile={profile} />}
+            subheader={<SubHeader profile={profile != null ? profile : student1} />}
```

Comment:
More shorthand for the same thing:
```suggestion
            subheader={<SubHeader profile={profile ?? student1} />}
```